.pre-commit-config.yaml

```diff
@@ -16,11 +16,11 @@ repos:
       - id: black
         args: ["--config", "python/cucim/pyproject.toml"]
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.1.3
+    rev: v0.6.1
     hooks:
       - id: ruff
         types_or: [python, pyi]
-        args: [--fix, --exit-non-zero-on-fix]
+        args: [--fix, --exit-non-zero-on-fix, --config, "python/cucim/pyproject.toml"]
```

Comment:
`ruff` in this repo was not actually reading the configuration at `python/cucim/pyproject.toml`, and there isn't another file with `ruff` configuration.

I assumed it was intentional that this would be read, based on https://github.com/rapidsai/cucim/pull/766#issuecomment-2304736099, so proposing this.

Since this configuration really applies to code across the entire repo, in the future you might want to consider moving this to a root-level `pyproject.toml` instead of one inside `python/cucim/`. That's what e.g. `cudf` does: https://github.com/rapidsai/cudf/blob/00ff2ee5ec2fd23c65e759dc2f9d2907a1c9cb00/pyproject.toml#L28